sort-out-dist.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Sort out the /dist folder, which is somewhat of a mess.
+# Ideally `make dist` would do this. But until then.
+cd ./dist || exit 1
+
+# now get a list of stuff
+STUFF=$(find icu-* -type f \( -name '*.zip' -o -name '*.tgz' \))
+
+trymove()
+{
+    base2=$(basename $1)
+    if [ -f ${2}/${3} ];
+    then
+        echo "# exists: ${2}/${3} (not copying $base2)"
+    else
+        mkdir -p ${2}
+        ln -v ${1} ${2}/${3}
+    fi
+}
+
+for file in ${STUFF};
+do
+    #echo "# ${file}"
+    base=$(basename $file)
+    # rver is '66-1'
+    rver=$(echo $file | grep "^icu-rrelease-[0-9][0-9]-[0-9]" | cut -d- -f3-4)
+    if [ "${rver}" = "" ];
+    then
+        echo "# ${file} - could not extract release version."
+        continue
+    fi
+    uver=$(echo $rver | tr - _)
+    prefix=icu4c-${uver}
+    dir=icu4c-$(echo $rver | tr - .)
+    mkdir -pv ./${dir}/
```

Comment:
Neither the `./` prefix nor the `/` suffix actually do or mean anything in this context.